src/clientManager.js

```diff
@@ -140,9 +140,10 @@ ClientManager.prototype.removeUser = function(name) {
 	return true;
 };
 
-ClientManager.prototype.autoload = function(/* sockets */) {
+ClientManager.prototype.autoload = function ClientAutoLoad() {
```

Comment:
I'm not sure we need this but if we do, common practice is to give it the same name than the callable name, so `function autoload() ...`.